cirq/linalg/distance_measures_test.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2020 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for distance_measures."""
+
+import numpy as np
+import pytest
+
+import cirq
+
+
+def test_fidelity():
+
+    # Known values
```

Comment:
I'd be nice to separate the test cases out into their own functions. This makes it easier to read (you don't have to read the full test), easier to see what failed/succeeded (with pytest -v) and enables better parallelization.